deploy/scripts/src/txma/test.ts

```diff
@@ -0,0 +1,62 @@
+import { type Options } from 'k6/options'
+
+import { selectProfile, type ProfileList, describeProfile } from '../common/utils/config/load-profiles'
+
+import { uuidv4 } from '../common/utils/k6/k6-utils'
+
+import { AWSConfig, SQSClient } from '../common/utils/aws/sqs'
+
```

Comment:
```suggestion
import { type Options } from 'k6/options'
import { selectProfile, type ProfileList, describeProfile } from '../common/utils/config/load-profiles'
import { uuidv4 } from '../common/utils/k6/k6-utils'
import { AWSConfig, SQSClient } from '../common/utils/aws/sqs'
```
Minor whitespace tidy up on the imports

Reply:
Done